doc/pages/example_notebooks/density/densitys_model_example.ipynb

```diff
@@ -4,11 +4,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# example for the implemented massdistributions of the Milky Way\n",
+    "# Example for the implemented mass distributions of the Milky Way\n",
```

Comment:
Maybe we should better speak of "gas density distributions", making clear that we are not referring to, e.g., the matter bound in stars.

Reply:
I agree, I've changed it.